plugins/outputs/prometheus_client/README.md

```diff
@@ -10,6 +10,16 @@ This plugin starts a [Prometheus](https://prometheus.io/) Client, it exposes all
   # Address to listen on
   listen = ":9273"
 
+  # Use TLS
+  tls = true
```

Comment:
Remove this option, if other tls options are set then enable tls, see http_listener input for an example.

Reply:
Done in https://github.com/influxdata/telegraf/pull/3719/commits/ffc0690b76d3b45a35c11f3efbf9dec4cc076e31